graphtool/graph.py

```diff
@@ -4,14 +4,20 @@ def __init__(self, data=None, neighbours=None):
         self.data = data
         if neighbours is None:
             self.neighbours = []
+        else:
+            self.neighbours = neighbours
         return self
 
-    def add_neighbours(self, neighbour):
+    def add_neighbour(self, neighbour):
```

Comment:
name of methods should be singular : good point

Reply:
Well, it is a syntax remark : this function adds ONE neighbour, so it's strange to call it `add_neighbours`